test/multipart_test.gleam

```diff
@@ -0,0 +1,83 @@
+import bucket/complete_multipart_upload
+import bucket/create_multipart_upload
+import bucket/get_object
+import bucket/upload_part
+import gleam/bit_array
+import gleam/httpc
+import gleam/list
+import gleam/string
+import gleeunit/should
+import helpers
```

Comment:
Please add tests for the various possible errors

Reply:
done in 3be6630